datasets/ONS-UK-trade-in-goods-by-industry-country-and-commodity/info.json

```diff
@@ -1,62 +1,71 @@
-            "rec6SvP3d4MJ3pXYR",
-            "recduvpZS53BpXU3W"
-        ],
-        "main_issue": 3,
-        "columns": {
-            "Period": {
-                "parent": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
-                "value": "http://reference.data.gov.uk/id/{period}",
-                "codelist": false
-            },
-            "Flow": {
-                "dimension": "http://gss-data.org.uk/def/trade/property/dimension/flow-directions",
-                "value": "http://gss-data.org.uk/def/trade/concept/flow-directions/{flow}"
-            },
-            "ONS Partner Geography": {
-                "dimension": "http://gss-data.org.uk/def/trade/property/dimension/ons-partner-geography",
-                "value": "http://gss-data.org.uk/def/concept/cl-area/{ons_partner_geography}"
-            },
-            "Commodity": {
-                "dimension": "http://gss-data.org.uk/def/trade/property/dimension/commodity",
-                "value": "http://gss-data.org.uk/def/trade/concept/commodity/{commodity}"
-            },
-            "Industry": {
-                "parent": "http://gss-data.org.uk/def/trade/property/dimension/industry"
-            },
-            "Value": {
-                "unit": "http://gss-data.org.uk/def/concept/measurement-units/gbp-million",
-                "measure": "http://gss-data.org.uk/def/measure/current-prices",
-                "datatype": "integer"
-            },
-            "Marker": {
-                "attribute": "http://purl.org/linked-data/sdmx/2009/attribute#obsStatus",
-                "value": "http://gss-data.org.uk/def/concept/cogs-markers/{marker}"
-            }
+    "main_issue": 3,
+    "columns": {
+      "Period": {
+        "type": "dimension",
+        "uri": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
+        "value": "http://reference.data.gov.uk/id/{period}"
+      },
+      "Flow": {
+        "type": "dimension",
+        "uri": "http://gss-data.org.uk/def/trade/property/dimension/flow-directions",
+        "value": "http://gss-data.org.uk/def/trade/concept/flow-directions/{flow}"
+      },
+      "ONS Partner Geography": {
+        "type": "dimension",
+        "uri": "http://gss-data.org.uk/def/trade/property/dimension/ons-partner-geography",
+        "value": "http://gss-data.org.uk/def/concept/cl-area/{ons_partner_geography}"
+      },
+      "Commodity": {
+        "type": "dimension",
+        "uri": "http://gss-data.org.uk/def/trade/property/dimension/commodity",
+        "value": "http://gss-data.org.uk/def/trade/concept/commodity/{commodity}"
+      },
+      "Industry": {
+        "type": "dimension",
+        "new": {
+          "codelist": "./codelists/industry.csv-metadata.json"
         }
-    },
-    "load": {
-        "publish": true
-    },
-    "sizingNotes": "",
-    "notes": ""
-}
+      },
+      "Value": {
+        "type": "observations",
+        "datatype": "double"
+      },
+      "Measure Type": {
```

Comment:
Just stumbled across this config here and wanted to say that this isn't the best way of doing single-measure datasets. 

If the measure & unit are the same for all measures, it's best not to have a measure/units column and to stick the measure/unit against the observation as [shown here](https://github.com/GSS-Cogs/gss-utils/blob/master/gssutils/csvcubedintegration/configloaders/ObservationsColumns.md#defining-observations-columns).